cobra/internal/base/moimpl.py

```diff
@@ -447,6 +483,31 @@ def __init__(self, parentMoOrDn, markDirty, *namingVals,
             # pylint:disable=protected-access
             self.__parentMo.__modifyChild(self, attach=True)
 
+    def clone(self, parentMo=None):
```

Comment:
This done for efficiency reasons. deep copy was very slow because it traverses the parent pointers as well and clones the whole world where in this case we just need subtrees.